Automattic/jetpack
# Protect: Cache empty results for a shorter period of time

#### Changes proposed in this Pull Request:

* Cache empty results for a shorter period of time
* Add tests :)
* Fix a small bug in get_all_vulnerabitlities()

#### Jetpack product discussion


1202064326332560-as-1202246145491819
#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Set your response type as `empty`
* Connect Protect and load the page
* Check your `jetpack_protect_status_time` option and verify it is a timestamp for only 5 minutes in the future 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202246145491819
